tests/system/providers/google/cloud/gcs/example_gcs_to_bigquery_async.py

```diff
@@ -0,0 +1,121 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example DAG using GCSToBigQueryOperator.
+"""
+from __future__ import annotations
+
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryDeleteDatasetOperator,
+)
+from airflow.providers.google.cloud.transfers.gcs_to_bigquery import GCSToBigQueryOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "gcs_to_bigquery_operator_async"
+
+DATASET_NAME_STR = f"dataset_{DAG_ID}_{ENV_ID}_STR"
+DATASET_NAME_DATE = f"dataset_{DAG_ID}_{ENV_ID}_DATE"
+TABLE_NAME_STR = "test_str"
+TABLE_NAME_DATE = "test_date"
+MAX_ID_STR = "name"
+MAX_ID_DATE = "date"
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
```

Comment:
Nit: You can move this os.environ to group them with other os.environ.get's (next to ENV_ID)

Reply:
Done